### ISSUE TITLE
"float" object is not callable, when running model.compile("adam", lr = 0.001)

### ISSUE THREAD

**Issue by arafim14** [lululxvi/deepxde#427]
Hello Dr. Lu,

I am trying to solve the modified Pennes Bioheat equation in 1D. I am getting this error. I am new to Deepxde, can you please help me how I can fix this error?

![deep1](https://user-images.githubusercontent.com/45234582/142021376-a689e27f-44aa-4a87-8c1c-e5f7437365ed.PNG)
![deep2](https://user-images.githubusercontent.com/45234582/142021429-6361dee5-b255-441e-9d68-82af4665ca2f.PNG)
![deep3](https://user-images.githubusercontent.com/45234582/142021439-0459084f-348f-449d-9c80-b7556fdc446a.PNG)
![deep4](https://user-images.githubusercontent.com/45234582/142021450-d13319ae-90e9-481a-bb2e-9c081fffd44b.PNG)

**Comment by lululxvi** [comment]
`bc1` is not correct. You need a function, not `y_c`.

**Comment by arafim14** [comment]
Thanks, I fixed it but the train loss, test loss are
![deep5](https://user-images.githubusercontent.com/45234582/142348532-66f21ce2-1b27-4f13-9365-5b50af1f6198.PNG)
 Nan values.

**Comment by lululxvi** [comment]
See FAQ.
